materials/dplyr.md

```diff
@@ -5,79 +5,102 @@ title: Working with Tabular Data
 language: R
 --- 
 
-## Packages
+> Remember to
+> 
+> * download [`surveys.csv`](https://ndownloader.figshare.com/files/2292172).   
+> * download [`portal_mammals.sqlite`](https://ndownloader.figshare.com/files/2292171).
+> * `setwd()` in R.
```

Comment:
Right, but they shouldn't be doing that they should be putting `surveys.csv` in their working directory by using an RStudio Project.